docs/userGuide/deployingTheSite.md

```diff
@@ -52,18 +52,44 @@ You can override the default deployment settings %%(e.g., repo/branch to deploy)
 ### Deploying to GitHub Pages via Travis CI
 **You can setup [<tooltip content="a platform for Continuous Integration and Delivery (among other things)">Travis CI</tooltip>](https://www.travis-ci.org/) to automatically build and deploy your site on GitHub Pages every time your GitHub repo is updated.**
 
-Here are the steps to set up Travis CI:
 
-1. [Sign up with GitHub](https://travis-ci.com/signin) at [Travis CI](https://travis-ci.com).
+#### Adding your repository to Travis CI
+
+<panel header="{{icon_info}} Travis CI Migration" type="info" expanded>
+
+As of May 2018, Travis CI was [undergoing migration to `travis-ci.com`](https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/), which changes the way you setup repositories on Travis CI. 
+
+- If you are new to Travis CI, or adding a new repository, you should **add your repository on `travis-ci.com`**.
+- If you are adding an existing repository, you should **add your repository on `travis-ci.org`**.
```

Comment:
The reader might wonder what exactly is an `existing repository`.

Reply:
I rewrote the instructions for this bit to advise users who are already using `travis-ci.org` to continue using it instead. While this may not be official Travis CI advice, I imagine it would be less confusing for a regular user to not have repositories scattered between two domains anyway.